index.html

```diff
@@ -2172,6 +2172,16 @@ <h2>Method Schemes</h2>
 formats as possible.
       </p>
 
+      <p>
+The <code>method-specific-id</code> format MAY include colons, which may be
+used by <a>DID methods</a> for various purposes, such as establishing
+hierarchically partitioned namespaces, or identifying specific instances or
+parts of the <a>DID registry</a>. However, any such use of the colon in the
+<code>method-specific-id</code> format is entirely method-specific, and
+<a>consumers</a> MUST NOT assume any meanings or behaviors associated with a
```

Comment:
```suggestion
implementers are advised to avoid assuming any meanings or behaviors associated with a
```

What was previously written isn't a testable statement, so removing the normative language.